.github/workflows/Docker-image-to-package.yml

```diff
@@ -0,0 +1,43 @@
+name: Create and publish a Docker image to ghcr.io
+
+on:
+  push:
+     branches: [ "main" ]
+  pull_request:
+     branches: [ "main" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
```

Comment:
Can you add Test, as we have tested 
Steps follow
- build
- test
- push to docker

Reply:
Sure. Will update you.